server/views/pages/crmDetails.njk

```diff
@@ -4,46 +4,50 @@
 {% from "../components/timeAndCost.njk" import timeAndCost %}
 {% from "../components/totalAndCost.njk" import totalAndCost %}
 
-{% set pageTitle = applicationName + " - Home" %}
+{% set pageTitle = applicationName + " - " + crmType + ": " + title %}
 {% set mainClasses = "app-container govuk-body" %}
 
 {% block content %}
 
     <div class="govuk-width-container">
         <main class="govuk-main-wrapper">
+            <span class="govuk-caption-l govuk-!-padding-bottom-1">eForm: {{ crmType }} | Case number: {{ usn }}</span>
             <h1 class="govuk-heading-l">{{ title }}</h1>
             <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-third">
+                <div class="govuk-grid-column-one-third ">
                     {{ navigationBar(navigationItems) }}
                 </div>
-                <div class="govuk-grid-column-two-thirds">
-                    <div>
-                        {% for subsection in section.subsections %}
-                        <div class="govuk-section-break govuk-section-break--l govuk-section-break--visible"></div>
-                        <div class="govuk-form-group">
-                            <h2 class="govuk-heading-l">{{ subsection.title }}</h2>
-                            {% for field in subsection.fields %}
-                                <div class="govuk-grid-row">
-                                    {% if field.subHeading %}
-                                        <div class="govuk-grid-column-full">
-                                            <h3 class="govuk-heading-m govuk-!-static-padding-top-5 govuk-!-static-margin-bottom-1">{{ field.subHeading }}</h3>
-                                        </div>
-                                    {% else %}
-                                        {% if field.type === 'timeAndCost' %}
-                                            {{ timeAndCost(field) }}
-                                        {% elif field.type === 'totalAndCost' %}
-                                            {{ totalAndCost(field) }}
+                <div class="govuk-grid-column-two-thirds govuk-!-padding-top-4">
+                    {% for subsection in section.subsections %}
+                        <div class="govuk-form-group govuk-summary-card govuk-!-margin-bottom-8">
+                            <div class="govuk-summary-card__title-wrapper">
+                                <h2 class="govuk-summary-card__title govuk-heading-l">{{ subsection.title }}</h2>
```

Comment:
I think there is a need to re-think the hierarchy of titles so we can have a visual difference in between `h1` / `h2` / `h3`. At the minute, `h1` and `h2` are fairly identical in size which can be visually confusing.

Reply:
I have made the h2 smaller